legate/settings.py

```diff
@@ -56,6 +56,17 @@ class LegateRuntimeSettings(Settings):
         """,
     )
 
+    full_bt_on_oom: PrioritizedSetting[bool] = PrioritizedSetting(
```

Comment:
Any way to avoid two initialisms at once? `full_trace_on_oom` or `full_oom_debug`?